src/drive/mobile/lib/network.js

```diff
@@ -20,9 +20,13 @@ const hasCordovaPlugin = () =>
 export const getConnectionType = () =>
   hasCordovaPlugin() ? navigator.connection.type : WIFI
 
-export const onWifi = connection => connection === WIFI
+const isWifi = connection => connection === WIFI
 
+// TODO: rename this function
+// There is no reason for a function in a `network.js` file
+// to be called something about `backup`...
```

Comment:
The name isn't tooo bad either, but maybe it should be somewhere else then?

Reply:
yes, that is true